### PR TITLE
74 update iojsonwebtokenjjwt

### PR DESCRIPTION
**Pull request by pwannenmacher**
close https://github.com/Frachtwerk/essencium-backend/issues/74

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Version ___
```
Comment (cr0ssing) [inline CHANGELOG.md:3]: Version number is missing

File: MIGRATION.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Migration Guide
 
+## Migrate to `___`
```
Comment (cr0ssing) [inline MIGRATION.md:3]: Version number is missing

**Review by cr0ssing — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pwannenmacher) [reply CHANGELOG.md:3]: The version number is still missing here on purpose. It is not yet certain whether the next release will be `2.4.6` or `2.5.0`. Before I push the corresponding tag the migration guide and changelog will be adjusted accordingly. The entry here is more a reminder...

File: MIGRATION.md (hunk 2 above)
Comment (pwannenmacher) [reply MIGRATION.md:3]: The version number is still missing here on purpose. It is not yet certain whether the next release will be `2.4.6` or `2.5.0`. Before I push the corresponding tag the migration guide and changelog will be adjusted accordingly. The entry here is more a reminder...

**Comment by pwannenmacher** [comment]
As long as we don't have a solution to persist a signing key over several Essencium instances in parallel (e.g. in load balancer scenarios), the secret must be set via an ENV variable. Therefore the latest changes.

**Review by cr0ssing — APPROVED** [review]

**Pull request merged**